Source/ZoomNet/Resources/Groups.cs

```diff
@@ -1,6 +1,44 @@
+using Pathoschild.Http.Client;
+using System.Text.Json.Nodes;
+using System.Threading;
+using System.Threading.Tasks;
+
 namespace ZoomNet.Resources;
 
-public class Groups
+/// <summary>
+/// Allows you to manage Groups.
+/// </summary>
+public class Groups : IGroups
 {
-	
+	private readonly IClient _client;
+
+	internal Groups(IClient client)
+	{
+		_client = client;
+	}
+
+	/// <summary>
+	/// Adds a user to a group.
+	/// </summary>
+	/// <param name="groupId">The ID of the group.</param>
+	/// <param name="userEmail">The email of the user to add to the group.</param>
+	/// <param name="cancellationToken">The cancellation token.</param>
+	/// <returns>A task representing the operation. The result will be true if the user was added successfully, false otherwise.</returns>
+	/// <inheritdoc/>
+	public async Task<bool> AddUserToGroupAsync(string groupId, string userEmail, CancellationToken cancellationToken = default)
+	{
+		var data = new JsonObject
+		{
+			{ "members", new JsonArray(new JsonObject { { "email", userEmail } }) }
+		};
+
+		var response = await _client
+			.PostAsync($"groups/{groupId}/members")
+			.WithJsonBody(data)
+			.WithCancellationToken(cancellationToken)
+			.AsMessage()
+			.ConfigureAwait(false);
+
+		return response.IsSuccessStatusCode;
```

Comment:
Get the group members identifiers:
```csharp
	// The sample response in Zoom's documentation looks like this: "ids": "v4iyWT1LTfy8QvPG4GTvdg"
	// I have no idea what it looks like when you add multiple users to a group.
	// Maybe it's a comma delimited string????
	return response.GetRawText().Split(',');  // This is just a guess on my part
```

Reply:
I tested this functionality with multiple users. It is indeed comma separated. Response looks like this:
```
{
  "ids": "v4iyWT1LTfy8QvPG4GTvdg,v3iyPT1LPfy8QvPF4GTvdi",
  "added_at": "2024-10-20T13:28:09Z"
}
```

Reply:
Excellent, I'm glad you verified.